src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -46,7 +46,6 @@ public override BigInteger TotalSupply(StoreView snapshot)
 
         protected override void OnBalanceChanging(ApplicationEngine engine, UInt160 account, NeoAccountState state, BigInteger amount)
         {
-            DistributeGas(engine, account, state);
```

Comment:
It probably breaks the logic. Say A has 10 NEO and B has 10 NEO, now I transfer 5 NEO from A to B. Even if A is the sender (which could easily be not the case, some C might pay for the transaction) GAS won't be distributed for B, although now it has a different balance (and what would be calculated for B now with its 15 NEO?).

Reply:
Better not change DistributeGas logic @shargon... just keep like this, and like @vncoelho mentioned, claim again if strictly necessary.

Reply:
Yes, removing this line breaks the logic of GAS disctribution.

Reply:
I will revert this change

Reply:
Done